README.md

```diff
@@ -28,31 +28,50 @@ By default all functions will then be automatically scheduled, wrapped to accept
 
 ## Options
 
-The plugin provides the following configurations.
+The plugin provides the following on a **per-function** configurations.
 
 | Option | Values | Default | Description  |
 | :--- | :--- | :--- | :--- |
 | `schedule` | Boolean | true | Control whether the lambda should be ignited every 5 minutes |
-| `functions` | mixed[], String or RegExp | ['/.*/'] | Which functions to perform wrapping, and immediate calls to post-deployment |
-
-#### Options Example
+| `name` | String or RegExp | '.*' | Which function name to perform wrapping, and immediate calls to during post-deployment |
+| `wrapper` | File Path | null | An override for the wrapper function, please include the extname |
```

Comment:
"File Path" is inconsistent with the other columns. Seems like it should be `String`. You can update the description to say the "The full file path, including extension, to a custom wrapper function."